.circleci/config.yml

```diff
@@ -27,11 +27,11 @@ jobs:
       - checkout
       - run:
           name: "Validate Orb Config"
-          command: circleci orb validate orb.yml
+          command: circleci config pack src > orb.yml && circleci orb validate orb.yml
```

Comment:
Might be worth upgrading this config to orb-tools v9, which has jobs and commands to more easily test/validate/deploy orbs.

Reply:
It's been a while since I've worked with orbs. I've added few things as per documentation. However I would appreciate if you have any examples.